tests/ts_A_select_compare_two_fields.erl

```diff
@@ -1,4 +1,4 @@
--module(ts_A_select_fail_5).
+-module(ts_A_select_compare_two_fields).
```

Comment:
This test name previously showed that this was negative testing. Reading the test name now it is not obvious whether two fields can be compared or not. `ts_A_select_compare_two_fields_not_allowed` would be more descriptive.